src/ray/gcs/gcs_client/global_state_accessor.cc

```diff
@@ -79,6 +79,19 @@ std::vector<std::string> GlobalStateAccessor::GetAllJobInfo() {
   return job_table_data;
 }
 
+int GlobalStateAccessor::GetNextJobID() {
+  std::promise<int> promise;
+  RAY_CHECK_OK(gcs_client_->Jobs().AsyncGetNextJobID(
+      [&promise](const int job_id) { promise.set_value(job_id); }));
+  auto future = promise.get_future();
+  auto status = future.wait_for(std::chrono::milliseconds(5000));
+  if (status == std::future_status::ready) {
+    return future.get();
+  } else {
+    throw std::runtime_error("Failed to get next job id");
```

Comment:
Will we catch this exception or let it fail?

Reply:
I'd like to change it to the same with other functions.

Reply:
done